src/renderers/WebGLRenderer.js

```diff
@@ -870,6 +870,8 @@ class WebGLRenderer {
 
 				if ( object._multiDrawInstances !== null ) {
 
+					// @deprecated, r174
+					warnOnce( 'THREE.WebGLRenderer: renderMultiDrawInstances has been deprecated and will be removed in r180. Append to renderMultiDraw arguments and use indirection.' );
```

Comment:
Should this be `r184` instead of `r180`? We normally plan for ten releases before deprecated code is removed.

Reply:
Yes! Fixed - thank you.